agr_literature_service/api/crud/topic_entity_tag_crud.py

```diff
@@ -586,6 +587,46 @@ def show_all_reference_tags(db: Session, curie_or_reference_id, page: int = 1,
         return [get_tet_with_names(db, tag, curie_to_name) for tag in all_tet]
 
 
+def get_reference_tags(db: Session, mod_abbreviation: str, days_updated: int = 7):
```

Comment:
Can we call the function `get_all_topic_entity_tags_by_mod` or something similar to make it clear that it's not by reference but for all tags in all references for the mod?

Reply:
Sounds good!

Reply:
fixed =)